src/ssl.c

```diff
@@ -6913,8 +6913,10 @@ WOLFSSL_ABI
 int wolfSSL_CTX_load_verify_locations(WOLFSSL_CTX* ctx, const char* file,
                                      const char* path)
 {
-    return wolfSSL_CTX_load_verify_locations_ex(ctx, file, path,
+    int ret = wolfSSL_CTX_load_verify_locations_ex(ctx, file, path,
         WOLFSSL_LOAD_VERIFY_DEFAULT_FLAGS);
+
+    return RETURN_CODE(ret,0);
```

Comment:
Please use WOLFSSL_FAILURE instead of 0 for alternate return code.  Applies to all RETURN_CODE sections in this PR where 0 is used.

Reply:
Replaced them all.

Reply:
Thanks